source/MRMesh/MROpenVDBHelper.h

```diff
@@ -0,0 +1,205 @@
+#pragma once
+#include "MRPch/MROpenvdb.h"
```

Comment:
please protect __EMSCRIPTEN__ and MRMESH_NO_VOXEL

Reply:
fixed